README.md

```diff
@@ -376,6 +377,21 @@ properties.pop("release")
 path.properties = properties
 ```
 
+## Repository Scheduled Replication Status ##
+Returns the status of scheduled  cron-based replication jobs define via the Artifactory UI on repositories.
+Supported by local, local-cached and remote repositories.
+Notes: Requires Artifactory Pro
+Security: Requires a user with 'read' permission (can be anonymous)
+```python
+from artifactory import ArtifactoryPath
+
+path = ArtifactoryPath("http://repo.jfrog.org/artifactory/distributions")
```

Comment:
Can we use full path here? `distributions` doesn't give a feeling that we should use "repo" here

```python
path = ArtifactoryPath("http://repo.jfrog.org/artifactory/repo-name/myfolder/deploy.1.1.23.zip")
path.replication_status

```

Reply:
I changed example, but I want to keep some running example, I took real URL